config/sidekiq.yml

```diff
@@ -2,6 +2,8 @@
 :queues:
   - default
   - stats
+  - creneaux
 
 :limits:
   stats: 2
+  creneaux: 1
```

Comment:
J'imagine qu'avec le changement le fait d'itérer sur toutes les `category_configurations` ne devrait pas prendre trop de temps, mais si jamais c'est le cas on pourra augmenter ce nombre (au moins à `2` ou `3`)